.pylintrc

```diff
@@ -115,10 +115,10 @@ evaluation=10.0 - ((float(5 * error + warning + refactor + convention) / stateme
 [BASIC]
 
 # Good variable names which should always be accepted, separated by a comma
-good-names=i,j,k,ex,Run,_,d,e,v,o,l,x,ts,f
+good-names=_,d,df,e,ex,f,i,id,j,k,l,o,pk,Run,ts,v,x
```

Comment:
nice!

Reply:
I think `e` should be a bad name. We should favor `ex` for exceptions.